docs/api-reference/next/font.md

```diff
@@ -109,19 +110,19 @@ Examples:
 
 The font [`display`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) with possible string [values](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display#values) of `'auto'`, `'block'`, `'swap'`, `'fallback'` or `'optional'` with default value of `'swap'`.
 
-Used in `@next/font/google` and `@next/font/local`
+Used in `next/font/google` and `next/font/local`
 
 - Optional
 
 Examples:
 
-- `display: 'swap'`: A string assigned to the `swap` value
+- `display: 'optional'`: A string assigned to the `optional` value
```

Comment:
swap is the default, let's keep this 👍

Reply:
Since `swap` is the default and the `display` prop is optional, display would be added if the user wants to use another option. `display: 'swap'` becomes redundant. 

We do mention above that `swap` is the default in the paragraph above (though the structure of this API page could be improved).